app/routes/wallet/credentials.py

```diff
@@ -109,14 +189,40 @@ async def get_credential_mime_types(
     return result
 
 
-@router.get("/{credential_id}/revocation-status", response_model=CredRevokedResult)
+@router.get(
+    "/{credential_id}/revocation-status",
+    response_model=CredRevokedResult,
+    summary="Get revocation status of a credential",
+)
 async def get_credential_revocation_status(
     credential_id: str,
     from_: Optional[str] = None,
     to: Optional[str] = None,
     auth: AcaPyAuth = Depends(acapy_auth_from_header),
 ) -> CredRevokedResult:
-    """Query the revocation status of a specific credential by ID."""
+    """
+    Query the revocation status of a specific credential by ID
+    ---
+
+    The status can be check on a specific time range by providing the `from_` and `to` parameters.
+    Pass the seconds, since Unix epoch, to the `from_` and `to` parameters to query the revocation
+    status for a specific time range.
```

Comment:
"The status can be check" -> The revocation status of a credential can be queried over a specific time range by passing unix timestamps to the `from_` and `to` parameters. Leaving these parameters blank will return the current revocation status.